examples/utilities.html

```diff
@@ -1428,6 +1428,242 @@ <h2 class="mb-3">Vertical Align</h2>
     </table>
 
 
+    <hr class="mt-5 mb-5">
+
+
+    <!--
+    ================================================
+    STATE UTILITIES
+    ================================================
+    -->
+
+    <h2 class="mb-3">Filters</h2>
+
+    <div class="row">
+      <div class="col-sm">
+        <code class="mb-2">.filter-sepia</code>
+        <img src="https://www.ucf.edu/wp-content/uploads/2017/03/College-Pages-Nav-Images-600x600-artshumanities.jpg" class="img-fluid filter-sepia">
```

Comment:
The examples are going to look crumby when the image files eventually are deleted from ucf.edu